eternalcore-core/src/main/java/com/eternalcode/core/translation/Translation.java

```diff
@@ -410,4 +410,30 @@ interface AutoMessageSection {
         Notice disabled();
     }
 
+    JailSection jailSection();
+
+    interface JailSection {
+        Notice jailLocationSet();
+        Notice jailLocationRemove();
+        Notice jailLocationNotSet();
+        Notice jailLocationOverride();
+
+        Notice jailDetainBroadcast();
+        Notice jailDetainPrivate();
+        Notice jailDetainCountdown();
+        Notice jailDetainOverride();
+
+        Notice jailReleaseBroadcast();
+        Notice jailReleaseSender();
+        Notice jailReleaseOffline();
+        Notice jailReleaseAll();
+        Notice jailReleaseNoPlayers();
+        Notice jailRelease();
+
```

Comment:
Can it be unified?  
jailDetainBroadcast
jailDetainPrivate
jailReleaseBroadcast
jailReleaseSender
jailRelease